src/App.tsx

```diff
@@ -1,26 +1,39 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadTodoFromServer, selectTodosLoaded } from './features/todos';
```

Comment:
The task description specifies that you should load the todos in the `App` on page load without using Redux Thunk. However, the `loadTodoFromServer` function is implemented using `createAsyncThunk`, which is part of Redux Thunk. Consider revising this approach to align with the task requirements .